### DISCUSSION TITLE
Request for Somali Language Support in Whisper ASR

### DISCUSSION THREAD

**Discussion by wehan1** [openai/whisper#2110] [Q&A]
Dear Whisper ASR Community,

I am writing to express my interest in adding Somali language support to the Whisper ASR system. I believe that including Somali would greatly enhance the accessibility and usability of this powerful tool.

Why Somali Matters:

Somali is spoken by millions of people worldwide, primarily in Somalia, Djibouti, and neighboring regions.
It plays a crucial role in communication, education, and cultural preservation.
By supporting Somali, Whisper can empower developers to create applications that cater to a broader audience.
Request: I kindly request that the Whisper ASR team consider adding Somali as a supported language. Whether through fine-tuning, custom vocabulary, or other approaches, this addition would be invaluable for developers working with Somali-speaking communities.

Community Collaboration: I am willing to contribute to this effort by providing Somali audio data, transcriptions, and any other necessary resources. Additionally, I encourage fellow developers and language enthusiasts to join this initiative.

Next Steps:

Let’s discuss the technical feasibility and requirements for Somali language integration.
Explore ways to collect and annotate Somali audio data for training purposes.
Collaborate on fine-tuning or adapting existing models to handle Somali speech.
I look forward to engaging with the community and contributing to this important endeavor. Together, we can make Whisper ASR even more inclusive and versatile.

Thank you for your attention, and I appreciate your commitment to advancing language technology.

Sincerely, Abdul, nurwehan@outlook.com

**Comment by glangford** [comment]
In case you have not seen this past discussion on Somali transcription, just FYI (no solution is offered unfortunately).

* https://github.com/openai/whisper/discussions/234

**Comment by Abokor7** [comment]
Have you received any help for this? Have you found any niche in the Somali community who is interested in making this to come to reality? If any, I would be interested in taking part ways to explore training existing TSS and STT models on the Somali language.

**Comment by wehan1** [comment]
Nope, that would be great.

**Reply by mate-ship-it** [reply]
Lads, I want to bring this back to life. Let's collaborate.

**Reply by mate-ship-it** [reply]
@wehan1 @Abokor7 I'm interested in training a small ASR and TTS model in Somali. I'd like to collaborate with you guys. Dadkeena way u baahan yihiin for us to do this. It can change the periphery of knowledge for our people. Hit me up guys.

**Comment by wehan1** [comment]
Sure! What’s a contact to reach you.

**Reply by mate-ship-it** [reply]
> Sure! What’s a contact to reach you.
> 
> On Tue, May 6, 2025 at 4:49 PM mate-ship-it ***@***.***>
> wrote:
> 
> > @wehan1 <https://github.com/wehan1> @Abokor7 <https://github.com/Abokor7>
> > I'm interested in training a small ASR and TTS model in Somali. I'd like to
> > collaborate with you guys. Dadkeena way u baahan yihiin for us to do this.
> > It can change the periphery of knowledge for our people. Hit me up guys.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/openai/whisper/discussions/2110#discussioncomment-13056100>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/AT2BSLQGJZ2OYUERFKX3ABT25FDA7AVCNFSM6AAAAABUGGM5HSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGMBVGYYTAMA>
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
> 

Contact me on +447437735154

**Reply by mate-ship-it** [reply]
> Sure! What’s a contact to reach you.
> 
> On Tue, May 6, 2025 at 4:49 PM mate-ship-it ***@***.***>
> wrote:
> 
> > @wehan1 <https://github.com/wehan1> @Abokor7 <https://github.com/Abokor7>
> > I'm interested in training a small ASR and TTS model in Somali. I'd like to
> > collaborate with you guys. Dadkeena way u baahan yihiin for us to do this.
> > It can change the periphery of knowledge for our people. Hit me up guys.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/openai/whisper/discussions/2110#discussioncomment-13056100>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/AT2BSLQGJZ2OYUERFKX3ABT25FDA7AVCNFSM6AAAAABUGGM5HSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGMBVGYYTAMA>
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
> 

Whatsapp me

**Reply by 5AHA1** [reply]
how can i join? do you have any whatsapp group?

**Comment by mate-ship-it** [comment]
Send me your number. I'll add you to a group

**Comment by moa14** [comment]
I am interested. I want to join

**Comment by wehan1** [comment]
elleven lab multilingual has great somali TTS.... look at it.

**Reply by moa14** [reply]
sxb speech to text baan rabay xagee ka helaa oo somali ah

**Reply by moa14** [reply]
linkden baan kaala soo xiriiray inaan haddii ay macquulthy i aqbal

**Reply by ak004** [reply]
sxb wax ma heshey ama wili waaye. if u found something let me i know i wanna contrubuite too.

**Reply by moa14** [reply]
https://www.linkedin.com/in/mohamed-ahmed-3b6ba420a?utm_source=share_via&utm_content=profile&utm_medium=member_android


Igala soo xiriir sxb

**Comment by lafano34** [comment]
Ok zpp baan u aqbala
